engine/src/main/java/org/terasology/rendering/opengl/FBO.java

```diff
@@ -279,6 +280,25 @@ private void setStatus(Status newStatus) {
     /**
      * Creates an FBO, allocating the underlying FrameBuffer and the desired attachments on the GPU.
      *
+     * Check FBO create(String title, Dimensions dimensions, Type type ...) for more.
+     * @param config A FBOConfig object that stores information used for creating FBO.
+     * @param dimensions A Dimensions object wrapping width and height of the FBO.
+     * @return The resuting FBO object wrapping a FrameBuffer and its attachments. Use getStatus() before use to verify completeness.
+     */
+    public static FBO create(FBOConfig config, FBO.Dimensions dimensions) {
```

Comment:
It's used for dynamic FBO's where config's Dimensions are irrelevant and scale factor is used.